guides/hosting/performance/lock-store.md

```diff
@@ -0,0 +1,21 @@
+# Lock store
+
+Shopware uses [Symfony's Lock component](https://symfony.com/doc/current/lock.html) to implement locking functionality.
+By default, Symfony will use a local lock store, this means in cluster setups the locking will only work on a single machine, therefore it is highly recommended using one of the [supported remote stores](https://symfony.com/doc/current/components/lock.html#available-stores).
+
+## Using Redis as a lock store
+
+As Redis can already be used for [caching](./caches.md), [increment store](./increment.md) and [session storage](./session.md), you can also use that Redis host as a remote lock store.
+To use Redis, create a `config/packages/framework.yml` file with the following content
+
+```yaml
+framework:
+    lock: 'redis://host:port'
+```
+
+## Other lock stores
+
+As Shopware uses [Symfony's Lock component](https://symfony.com/doc/current/lock.html) all lock stores supported by Symfony can be used.
```

Comment:
Can you use our specific Symfony version in the links?

Reply:
Done :heavy_check_mark: 

when we update the symfony version we should also update the symfony links